src/components/create-filter-form.js

```diff
@@ -221,7 +300,7 @@ const CreateFilterDialog = ({
                     variant="outlined"
                     size="small"
                     onClick={handleSave}
-                    disabled={disableBtnSave}
+                    disabled={!filterNameChecked || filterInvalid}
```

Comment:
`filterNameChecked` and `filterInvalid` represent the same value but inverted. I think we could use only one for now.